### PR TITLE
start 5.4.1 changelog

### PR DESCRIPTION
**Pull request by ashnwade**
This PR addresses no issue. Updating the changelog and training pdf for 5.4.1

### CODE REVIEW

**Comment by ashnwade** [comment]
please also compare with the list on the project board: https://github.com/orgs/gravwell/projects/1/views/39

File: changelog/5.4.1.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+* Fixed an issue that caused the cursor to jump around when editing a variable in a Template query.
+* Fixed an issue that allowed assets to be created with a whitespace-only name.
+* Fixed an issue that allowed invalid characters to be typed in for Font Size preferences.
+* Prevented passwords from being auto-populated on user forms and preferences.
+
+## Backend Changes
+
+### Bug Fixes
+
+* Fixed an issue with RMP package missing postinstall service restart command.
```
Comment (david-fritz-gravwell) [inline changelog/5.4.1.md:28]:
```suggestion
* Fixed an issue with some RPM packages missing postinstall service restart command.
```

**Review by david-fritz-gravwell — CHANGES_REQUESTED** [review]

**Review by michael-wisely-gravwell — COMMENTED** [review]
UI changes LGTM :+1:

**Review by david-fritz-gravwell — APPROVED** [review]
LGTM

**Pull request merged**